prow/spyglass/lenses/podinfo/podinfo_test.go

```diff
@@ -109,24 +110,37 @@ func TestBody(t *testing.T) {
 }`),
 				},
 			},
-			ownConfig: ownConfig{},
+			ownConfig: nil,
 		},
 	}
 
+	logHook := test.NewGlobal()
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
+			logHook.Reset()
 			wantFile := path.Join("testdata", "test_"+tc.name+".html")
 			wantBytes, err := ioutil.ReadFile(wantFile)
 			if err != nil {
 				t.Fatalf("Failed reading output file %s: %v", wantFile, err)
 			}
-			oc, err := json.Marshal(tc.ownConfig)
-			if err != nil {
-				t.Fatal(err)
+			var oc []byte
+			if tc.ownConfig != nil {
+				oc, err = json.Marshal(tc.ownConfig)
+				if err != nil {
+					t.Fatal(err)
+				}
 			}
 			got, want := Lens{}.Body(tc.artifacts, ".", "", json.RawMessage(oc), config.Spyglass{}), string(wantBytes)
 			if got != want {
-				t.Fatalf("Output mismatch\nwant: %s\n got: %s", want, got)
+				t.Errorf("Output mismatch\nwant: %s\n got: %s", want, got)
+			}
+			if entries := logHook.AllEntries(); len(entries) > 0 {
```

Comment:
I guess we're just checking that we did not emit any logs?

Reply:
Correct, the code path we are testing should not emit any logs if everything is working. If we later add debug or info level logs that should not count as failures we can update the unit test accordingly.